pkg/api/validation/validation_test.go

```diff
@@ -8609,6 +8609,71 @@ func getStorageResourceList(storage string) api.ResourceList {
 	return res
 }
 
+func getLocalStorageResourceList(ephemeralStorage string) api.ResourceList {
+	res := api.ResourceList{}
+	if ephemeralStorage != "" {
+		res[api.ResourceEphemeralStorage] = resource.MustParse(ephemeralStorage)
+	}
+	return res
+}
+
+func TestValidateLimitRangeForLocalStorage(t *testing.T) {
+	successCases := []struct {
```

Comment:
Is there errorCases?

Reply:
No, i copied it from `TestValidateLimitRange`, and intended to test local ephemeral storage feature gate here

Reply:
Modified cases name